### ISSUE TITLE
Navbar Icons don't display correctly on mobile if they are too wide.

### ISSUE THREAD

**Issue by Dr-Ari-Gami** [LazeMSS/OctoPrint-UICustomizer#197]
With just a few plugins installed that add navbar indicators, on mobile they run off the screen. It would be nice if if they horizontally scrolled. 

![Example Screenshot of Issue](https://cloud.gami.app/cloud/index.php/s/wFHTMifx5SKsaLk/download "Example Screenshot of Issue")

I'd submit a pr, but It's been a few years since I've done web development.

**Comment by LazeMSS** [comment]
I had it working in a release but it caused problems with plugins overflowing with popovers etc.

I'm still working on a solution

**Comment by electronicm** [comment]
Would it make sense to determine the height of the div via JS and move all content downwards accordingly?

What I could well imagine would be the same mechanism as is used for the tabs in the main section. So that all icons that exceed the display width are in a dropdown.

However, I see the display of the DisplayLayerProgress plugin as a fundamental problem. Due to the display of long text, this is not suitable for small display areas. Should the developer be involved in solving the problem?

**Comment by LazeMSS** [comment]
Yes but it would require a resize event handler - I did 90% of the work in CSS and a bit of js - but hiding the overflow caused the popovers etc. for some plugins to be hidden.

I think I will do a complete rewrite where its handle ind a better way to have the icons in a pull down fasion - still testing a lot of ideas.

**Comment by NilsRo** [comment]
Perhaps you can do the same and put all LI elements into a dropdown. Would not looks that nice but the UI will stay accessible.
![IMG_0161](https://user-images.githubusercontent.com/51311453/210550704-8b1101f0-fa47-41ca-af43-1a8360affcaf.jpg)
